### ISSUE TITLE
In standalone library regl is named initREGL

### ISSUE THREAD

**Issue by Erkaman** [mikolalysenko/regl#236]
If you export the standalone version `regl.min.js` as:

```
<script language=javascript src="regl.min.js"></script>
```

Then the regl constructor is made into a global variable. But for some reason, it is named `initREGL` instead of the traditional `regl`. Why is that?

**Comment by WebSeed** [comment]
Doesn't it have to be named differently in order to avoid a collision between the factory / creation function for `regl` and the instance name e.g. `const regl = initREGL()`. ES6 modules has same issue, have to do something like `import createREGL from 'regl'; const reg = createREGL()`.

**Comment by Erkaman** [comment]
Hmm, maybe you're right. But I also want to hear @mikolalysenko's answer.

**Comment by mikolalysenko** [comment]
WebSeed has it exactly right.

`initREGL` is a constructor.  Normally `module.exports` from the script tag gives you the *constructor* of the `regl` instance, but because it is an anonymous closure it doesn't have any particular name.  In the case of using a script tag we need to assign it to some global variable, so I just picked `initREGL`.  

That said, this should be documented better.

**Comment by Erkaman** [comment]
A problem right now is that even if you import regl.js from npm cdn:

```
<script src="https://npmcdn.com/regl/dist/regl.js"></script>
```

you can't find `initREGL ` nor `regl`. So we'll need to update regl on npm cdn.

**Comment by WebSeed** [comment]
I think the issue is that the build scripts are slightly inconsistent. Looks like in this case `reglInit` is the global, not `initREGL`.

`build-script` step
https://github.com/mikolalysenko/regl/blob/gh-pages/package.json#L71
```json
"build-script": "browserify regl.js --standalone reglInit > dist/regl.js",
```

`build-min` step
https://github.com/mikolalysenko/regl/blob/gh-pages/package.json#L72
https://github.com/mikolalysenko/regl/blob/gh-pages/bin/build-min.js#L13
```
browserify(INPUT_FILE, { standalone: 'initREGL' })
```

I'd raise a PR but not sure on your preferred naming.

**Comment by mikolalysenko** [comment]
Let's call it `createREGL` just like we do in all the test cases and do it that way.

Thanks for catching this, I don't really use the standalone scripts much so these things can slip through.

**Comment by Erkaman** [comment]
Yeah, let's call it that.

**Comment by mikolalysenko** [comment]
@WebSeed if you send a PR, I'll merge it in right away

**Comment by mikolalysenko** [comment]
Fixed in #241

**Comment by Erkaman** [comment]
Also, you should probably document in the README that it is called `createREGL `.
